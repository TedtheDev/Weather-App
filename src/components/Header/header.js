import React from 'react';
import styled from 'styled-components';

const HeaderDiv = styled.header`
    grid-area: header;
    display: flex;
    justify-content: center;
    align-items: center;
    font-size: 1.4rem;
    background: #009494;
    color: #F5F2DC;
    box-shadow: 0 0 2px #000;
`;

const Header = props => {
    return (
        <HeaderDiv>Five Day Forecast</HeaderDiv>
    );
}

export default Header;