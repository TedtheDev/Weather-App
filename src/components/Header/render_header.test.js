import React from 'react';
import Header from './header';
import renderer from 'react-test-renderer';

describe('<Header />', () => {
    test('should render correctly', () => {
        const component = renderer.create(
            <Header />
        );

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});