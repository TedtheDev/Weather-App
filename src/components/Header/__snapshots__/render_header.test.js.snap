// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<header
  className="sc-bdVaJa fctiBP"
>
  Five Day Forecast
</header>
`;
